examples/edge-stack-iso/custom/files/edge-stack-setup.service

```diff
@@ -0,0 +1,29 @@
+[Unit]
+Description=Setup Edge stack components
+Wants=network-online.target
+# It requires rke2 or k3s running, but it won't fail if those services are not present
+After=network.target network-online.target rke2-server.service k3s.service
+# At least, the common.sh needs to be present
+ConditionPathExists=/opt/edge/bin/basic-setup.sh
+
+[Service]
+User=root
+Type=forking
+# Metal3 can take A LOT to download the IPA image
+TimeoutStartSec=1800
+
+ExecStartPre=/bin/sh -c "echo 'Starting to setup Edge components'"
+# Scripts are executed in StartPre because it can be multiple ones
```

Comment:
It can be just a single 'ExecStart' per systemd service file, so instead of having multiple services, the scripts are executed as 'pretasks' then the start is just an echo.

Reply:
Can we reword this to `Scripts are executed in StartPre because Start can only run a single one`?